tests/jq.test

```diff
@@ -86,6 +87,10 @@ null
 "\u03bc"
 "%CE%BC"
 
+@urid
```

Comment:
Please also add test for error paths

Reply:
sorry i totally missed that there was a base64.test file! i just added a uri.test file with more extensive testing

Reply:
Hmm, fine if you instist, I guess.